site/src/pages/components/switch.explainer.mdx

```diff
@@ -0,0 +1,448 @@
+| `thumb`               | Enables styling the `thumb` part without requiring it to be completely replaced via its slot. |
+| `track`               | Enables styling the `track` part without requiring it to be completely replaced via its slot. |
+| `active-track-side`   | Enables styling the track side visible in the "on" state.                                     |
+| `inactive-track-side` | Enables styling the track side visible in the "off" state.                                    |
+
+---
+
+## Behavior
+
+### States and Interactions
+
+{/* _List key component states, valid state transitions, and how interactions trigger state transitions._ */}
+
+{/* - _Are states changed as a result of keyboard? mouse? touch?_ */}
+{/* - _Are there corresponding keyboard interactions for mouse/touch interactions?_ */}
+{/* - _Are some events triggered on key down vs key press?_ */}
+
+<div class="question">
+  Should the switch value type be a boolean or the string set ["on", "off"]?
+</div>
+
+| State Group | States            | Initial State | Description                                                                                                           | Interaction/Transition                                                                                      |
+| ----------- | ----------------- | ------------- | --------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------- |
+| `value`     | `"on"`/`"off"`(?) | `"off"`       | Determines the current value of the component.                                                                        | Changed upon user interaction on click, touch, space key (when focused), enter key (when focused, optional) |
+| `disabled`  | `true`/`false`    | `false`       | When `true`, disables the control, preventing user interaction and displaying the control with a disabled appearance. | No interaction. Controlled programmatically.                                                                |
+| `focused`   | `true`/`false`    | `false`       |                                                                                                                       |                                                                                                             |
+
+### Accessibility
+
+{/* - A labelling text can be assigned to the switch like any other form control through a `<label>` element, the `aria-label=""` attribute or the `aria-labelledby=""` attribute */}
+{/* - Should be assigned the implicit aria role of "switch" */}
+{/* - The current value should be reported as "on" or "off" to screen readers */}
+
+ARIA authoring practices guide for the switch pattern: https://www.w3.org/WAI/ARIA/apg/patterns/switch/
+
+<div class="question">
+  How can the `<switch>` element be assigned the implicit role of "switch"?
```

Comment:
if it became an actual element - it would be defined in HTML AAM. and HTML AAM would point to the Core AAM definition for the switch role mappings, here - https://w3c.github.io/core-aam/#role-map-switch